xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/action-menus.less

```diff
@@ -151,6 +151,28 @@
   }
 }
 
+#tmNotifications {
```

Comment:
The ``#tmNotifications`` HTML element is generated by a UIX defined in a wiki page, ``XWiki.Alerts.Code.AlertsMenuUIX``, so I'd prefer to keep the styles in the same place (to be packaged / installed / uninstalled together), e.g. using a SSX object on that page with LESS enabled.

Reply:
As there is already style related to the notification header in `XWiki.Notifications.Code.NotificationsDisplayerUIX`, I'll change the CSS SSX in here to a LESS and move my new style here :)
Note: There's also the javascript dealing with the behavior of `#tmNotifications` here